src/components/offers-list/offers-list.tsx

```diff
@@ -16,12 +16,16 @@ function OffersList({offers}: OffersListProps) {
   const handleMouseEnter = (id: string) => setActiveItem(id);
   const handleMouseLeave = () => setActiveItem('');
 
+  const points = offers.map((item: OfferListItem) => ({...item.location}));
```

Comment:
Лишний спред в цикле) так лучше не делать
```suggestion
  const points = offers.map((item: OfferListItem) => ({...item.location}));
```
```suggestion
  const points = offers.map(({ location, id }: OfferListItem) => ({ location, id}));
```